packages/trader/src/Stores/Modules/SmartChart/Helpers/barriers.ts

```diff
@@ -0,0 +1,18 @@
+import type { ChartBarrierStore } from '../chart-barrier-store';
+
+export type TBarrier = ChartBarrierStore & { key?: string };
```

Comment:
for me i ended up adding key as a prop in chartbarrierstore in my PR

Reply:
@henry-deriv the thing is that ChartBarrierStore has nothing to do with the key, the key is assigned afterwards to a resulting object which is returned as a result of `new ChartBarrierStore(...)` call.

Reply:
I made the same change in my PR

Reply:
thank you🙏🏻